decidim-core/app/helpers/decidim/scopes_helper.rb

```diff
@@ -41,6 +41,21 @@ def scopes_picker_field(form, name, root: false, options: {})
       end
     end
 
+    # Renders a scopes picker field in a form, not linked to a specific model.
+    # name - name for the input
+    # value - value for the input
+    #
+    # Returns nothing.
+    def scopes_picker_tag(name, value, options: {})
```

Comment:
I'm sorry to say this method isn't tested :trollface:

Reply:
Yeah, I know. I wish you uninstalled that codecov plugin xD. I'll work on it :)

Reply:
hahaha never!